run_gui.py

```diff
@@ -18,7 +20,14 @@
 
 app = QApplication(sys.argv)
 theme_picker(app, args.theme)
-a = App([os.path.join(data_path, 'example-streams', '1.mp4'), os.path.join(data_path, 'example-streams', '2.mp4'),
-        os.path.join(data_path, 'example-streams', '1.mp4'), os.path.join(data_path, 'example-streams', '2.mp4')])
+a = App([
+        VideoSource(os.path.join(data_path, 'example-streams', '1.mp4'), cv2.CAP_FFMPEG),
+        VideoSource(os.path.join(data_path, 'example-streams', '2.mp4'), cv2.CAP_FFMPEG),
+        VideoSource(os.path.join(data_path, 'example-streams', '1.mp4'), cv2.CAP_FFMPEG),
+        VideoSource('udpsrc port=5600 caps = "application/x-rtp, media=(string)video, clock-rate=(int)90000, encoding-name=(string)H264, payload=(int)96" ! rtph264depay ! decodebin ! videoconvert ! appsink', cv2.CAP_GSTREAMER)
```

Comment:
Maybe we should not open with the gstreamer by default since that seems to cause problems if the gazebo simulation isn't running.